pallets/phala/src/stakepool.rs

```diff
@@ -150,6 +150,11 @@ pub mod pallet {
 		T::MiningEnabledByDefault::get()
 	}
 
+	/// Helper storage to track the preimage of the mining sub-accounts. Not used in consensus.
+	#[pallet::storage]
+	pub type SubAccountPreimages<T: Config> =
+		StorageMap<_, Twox64Concat, T::AccountId, (u64, WorkerPublicKey)>;
```

Comment:
Do you think we can `(u64, WorkerPublicKey) => T::AccountId`, because StakePool extrisics always require pid and worker, by this design we can also avoid computing when it need to get the Miner account

Reply:
Fine, after all it's only used off-chain. However in any case as long as you have (pid + worker pubkey), you should always compute the miner account than reading it from the database or event the node, because the later case is way more expensive in both IO and CPU